dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -33,6 +34,7 @@
  *
  * @author dyuen
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
On the other hand, this is not for talking to quay.io. 
This is our own web service. 

Why is this being modified when the issue is communication with quay.io?
What is the unknown property that needs to be ignored when serializing and deserializing WorkflowVersions?